package.json

```
@@ -146,7 +146,7 @@
     "react-native-linear-gradient": "^2.8.1",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "2.0.10",
+    "react-native-onyx": "hotfix-2.0.10",
```

does this work or does it need to be like this?

```suggestion
    "react-native-onyx": "github:Expensify/react-native-onyx.git#hotfix-2.0.10",
```

---

Isn't it strange that `npm i` didn't throw an error about this? That's what made me think my change was OK, but let me check the source files

---

I think you are right though. I'm trying it out and verifying the changes